website/source/install.rst

```diff
@@ -18,9 +18,18 @@ An example setup including configuration and Kafka connection is available as co
 
 Then you should be able to reach two sets of endpoints:
 
-* Karapace schema registry on http://localhost:8081
+* Karapace schema registry master on http://localhost:8081
```

Comment:
Do you mean https here for intercommunication between master/slaves ?

Reply:
> Do you mean https here for intercommunication between master/slaves ?

you mean master/follower 😅 😉 ?
Yes exactly that, with mTLS support, we need now to use the proper scheme on the URL.

Reply:
I've been thinking to also use the [HTTPSRedirect](https://fastapi.tiangolo.com/advanced/middleware/#httpsredirectmiddleware) middleware from FastAPI to redirect all `http` requests. 
This might be nice to have, but we need to verify the behaviour of the redirect as we already have some request forwarding in place.

Reply:
```suggestion
* Karapace schema registry master on https://localhost:8081
```

Oh, my question was about protocol. Should this be https ?

Reply:
That is left as is, because there is still the option to have the service running without mTLS, thus then the scheme/protocol will be `http`, so to help, I added the make target `curl-sr-https` which is used to interface with the API on mTLS or noTLS and I then added an instruction for this in the docs.